Atos_test.py

```diff
@@ -0,0 +1,20 @@
+import random
```

Comment:
Jeśli ten skrpyt wykonuje jakieś konkretne zadanie , w tym przypadku (Skrypt wykonujący 'GuessNumber') nazwa pliku powinna to odwzierciedlać. Usunąć komentarze, zmienić nazwe pliku